spec/factories.rb

```diff
@@ -130,6 +130,7 @@
     sequence(:title) { |i| "title - #{i}" }
     sequence(:base_path) { |i| "link - #{i}" }
     sequence(:description) { |i| "description - #{i}" }
+    content_json nil
```

Comment:
I'm not sure this field needs to be populated by the factory at all. We want it to be nil until is has been populated in the spec we are writing. I just put in in there so it would be explicit but now you've pointed it out, it would be better to be explicit in the spec itself.

Reply:
Yes, this is a good point Pete. Better to be explicit in the spec